src/main/content/_includes/head.html

```diff
@@ -26,11 +26,11 @@
   {% css openliberty %}
   {% css coderay-custom %}
 
-  {% for css_name in page.css %}
+  {% for css_name in layout.css %}
     {% css {{css_name}} %}
   {% endfor %}
-  
-  {% for css_name in layout.css %}
+
+  {% for css_name in page.css %}
```

Comment:
Were these flipped so that one of the css files could override another?

Reply:
I flipped these so that the header.css would load before the css for the individual page. That way the specific page's css would override the general css from header.css. This was so that I could customize the nav link colors on the support and downloads pages.